providers/common/include/prov/fipsindicator.h

```diff
@@ -52,8 +52,8 @@
  * settable.
  */
 typedef struct ossl_fips_ind_st {
-    unsigned int approved;
-    int settable[OSSL_FIPS_IND_SETTABLE_MAX]; /* See OSSL_FIPS_IND_STATE */
+    unsigned char approved;
+    signed char settable[OSSL_FIPS_IND_SETTABLE_MAX]; /* See OSSL_FIPS_IND_STATE */
 } OSSL_FIPS_IND;
```

Comment:
Is this change of ABI going to be a problem for anything?

Reply:
This is a new thing. Also it is an internal structure, not public.